web/yo/app/scripts/controllers/gene.js

```diff
@@ -360,6 +363,7 @@ angular.module('oncokbApp')
                 var currentReviewer = $scope.realtimeDocument.getModel().createString(User.name);
                 $scope.gene.name_review.set('currentReviewer', currentReviewer);
                 $scope.allMyChanges = false;
+                $scope.savingAll = false;
```

Comment:
We have an object to store all status call `$scope.status`. Could you move to that?